core/test/integration/api/api_posts_spec.js

```diff
@@ -18,9 +16,9 @@ describe('Post API', function () {
     var localSettingsCache = {};
 
     before(testUtils.teardown);
-    after(testUtils.teardown);
+    afterEach(testUtils.teardown);
 
-    before(testUtils.setup('users:roles', 'perms:post', 'perms:init', 'posts'));
+    beforeEach(testUtils.setup('users:roles', 'perms:post', 'perms:init', 'posts'));
```

Comment:
We now reset the whole db after each test. Isn't it enough to reset the db after the test case you have added? 🤔

Reply:
Good :fishing_pole_and_fish: Tested without `Each` on current suite and seems to work. Reverted this change

Reply:
👍